src/app/public/modules/radio/fixtures/radio-group-reactive.component.fixture.html

```diff
@@ -1,10 +1,10 @@
 <form
   [formGroup]="radioForm"
 >
-  <sky-radio-group *ngIf="radioGroupEnabled"
+  <sky-radio-group *ngIf="showRadioGroup"
     class="form-group radio"
-    formControlName="option"
-    name="option"
+    formControlName="radioGroup"
+    name="radioGroup"
```

Comment:
☝️ updated the control name and element name to be more semantic. This isn't really an `option`, but instead the entire radio group.